bin/print_targets.ml

```diff
@@ -0,0 +1,27 @@
+open! Stdune
+open Import
+open Fiber.O
+
+let doc = "Print available targets"
+
+let man =
+  [ `S "DESCRIPTION"
+  ; `P {|$(b,dune targets) print the available targets|}
+  ; `Blocks Common.help_secs
+  ]
+
+let info = Term.info "targets" ~doc ~man
+
+let term =
+  let%map common = Common.term in
+  Common.set_common common ~targets:[] ;
+  let log = Log.create common in
+  Scheduler.go ~log ~common (fun () ->
+      Import.Main.setup ~log common ~external_lib_deps_mode:true
```

Comment:
is the `external_lib_deps_mode` part intentional? not entirely sure what it does, but that's not the default value so I'm wondering if this is necessary.